projects/plugins/jetpack/modules/masterbar/admin-menu/class-atomic-admin-menu.php

```diff
@@ -388,17 +388,17 @@ public function add_tools_menu() {
 		parent::add_tools_menu();
 
 		/**
-		 * Whether to show the WordPress.com Site Logs submenu under the main Tools menu.
+		 * Whether to show the WordPress.com Site Monitoring submenu under the main Tools menu.
 		 *
-		 * @use add_filter( 'jetpack_show_wpcom_site_logs_menu', '__return_true' );
+		 * @use add_filter( 'jetpack_show_wpcom_site_monitoring_menu', '__return_true' );
 		 * @module masterbar
 		 *
 		 * @since 12.0
 		 *
-		 * @param bool $show_wpcom_site_logs_menu Load the WordPress.com Site Logs submenu item. Default to false.
+		 * @param bool $show_wpcom_site_monitoring_menu Load the WordPress.com Site Monitoring submenu item. Default to false.
 		 */
-		if ( apply_filters( 'jetpack_show_wpcom_site_logs_menu', false ) ) {
-			add_submenu_page( 'tools.php', esc_attr__( 'Site Logs', 'jetpack' ), __( 'Site Logs', 'jetpack' ), 'manage_options', 'https://wordpress.com/site-logs/' . $this->domain, null, 7 );
+		if ( apply_filters( 'jetpack_show_wpcom_site_monitoring_menu', false ) ) {
```

Comment:
If we merge this PR _after_ we launch the feature via wpcomsh, then this filter isn't necessary.

Reply:
Having this filter leaves a way to enable/disable that link on `wpcomsh` side, for example, if we would like to enable it only for particular A12s. Do we assume that at the point of Jetpack release, we want to enable it for all users?

Reply:
> Do we assume that at the point of Jetpack release, we want to enable it for all users?

@wojtekn Yes. We're launching the feature with wpcomsh. We'll merge the Jetpack PR afterwards, and then remove the wpcomsh code once Jetpack is deployed to Atomic.